gallery_dl/extractor/steamgriddb.py

```diff
@@ -56,16 +56,16 @@ def items(self):
         download_fake_png = self.config("download-fake-png", True)
 
         for asset in self.assets():
-            urls = [asset["url"]]
+            urls = (asset["url"],)
             if download_fake_png and asset.get("fake_png"):
-                urls.append(asset["fake_png"])
+                urls = (asset["url"], asset["fake_png"])
```

Comment:
This is not exactly what I meant, but I guess it's my fault for not including a full suggestion.

``` python
            if download_fake_png and asset.get("fake_png"):
                urls = (asset["url"], asset["fake_png"])
            else:
                urls = (asset["url"],)
```